flujo/utils/context.py

```
@@ -60,6 +60,12 @@ def get_excluded_fields() -> set[str]:
         _ENV_EXCLUDED_FIELDS_CACHE = os.getenv("EXCLUDED_FIELDS", "")
 
     excluded_fields = _ENV_EXCLUDED_FIELDS_CACHE
+    # Defensive: ensure excluded_fields is a string-like value
+    if not isinstance(excluded_fields, str):  # pragma: no cover - CI hardening
+        try:
+            excluded_fields = str(excluded_fields) if excluded_fields is not None else ""
+        except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)